src/web/calculators/flywheel/index.ts

```diff
@@ -5,7 +5,7 @@ import PageConfig from "common/models/PageConfig";
 import { MeasurementParam, MotorParam, RatioParam } from "common/models/Params";
 import Ratio, { RatioType } from "common/models/Ratio";
 import { lazy } from "react";
-import { BooleanParam, withDefault } from "serialize-query-params";
+import {BooleanParam, NumberParam, withDefault} from "serialize-query-params";
```

Comment:
you can run `yarn dev:format` to format code using Prettier (or Prettier's vscode extension has a formatter tool).